atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/CassandraSchemaLockTest.java

```diff
@@ -82,6 +86,10 @@ public void shouldCreateTablesConsistentlyWithMultipleCassandraNodes() throws Ex
             assertTrue(executorService.awaitTermination(4, TimeUnit.MINUTES));
         }
 
+        CassandraKeyValueService kvs =
+                CassandraKeyValueService.create(configManager, Optional.absent());
+        assertThat(kvs.getAllTableNames(), hasItem(table1));
```

Comment:
When we were trying to run the test in a parameterised way, the tests were being kicked off before the old containers were fully shut down and the new ones started up. This test would then appear to pass very quickly, when in fact the container wasn't ready yet.

In short, this extra clause tests that we actually managed to create at least one table, which sounds reasonable to me.